homeassistant/components/lektrico/sensor.py

```diff
@@ -0,0 +1,296 @@
+"""Support for Lektrico charging station sensors."""
+
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from typing import Any
+
+from lektricowifi import Device
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_FRIENDLY_NAME,
+    PERCENTAGE,
+    UnitOfElectricCurrent,
+    UnitOfElectricPotential,
+    UnitOfEnergy,
+    UnitOfPower,
+    UnitOfTemperature,
+    UnitOfTime,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import PlatformNotReady
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import LektricoDeviceDataUpdateCoordinator
+from .const import DOMAIN
+from .entity import LektricoEntity
+
+
+@dataclass(frozen=True, kw_only=True)
+class LektricoSensorEntityDescription(SensorEntityDescription):
+    """A class that describes the Lektrico sensor entities."""
+
+    value_fn: Callable[[Any], float | str | int]
```

Comment:
Can we use a better type than Any? This is now completely messing up the typesafety of the sensors and is causing a lot of extra object creation because of all the `int()` and `str()`

Reply:
I changed it, thanks.